### PR TITLE
Feature/BountyHunterClass

### PR DESCRIPTION
**Pull request by soukunsandesu**
Dictionary: isTargetKilled削除
不要な変数削除

わからないところはなんでも質問してください。

### CODE REVIEW

File: Roles/BountyHunter.cs [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+                ResetTarget(killer);
+            }
+            else
+            {
+                Logger.Info($"{killer?.Data?.PlayerName}:ターゲット以外をキル", "BountyHunter");
+                Main.AllPlayerKillCooldown[killer.PlayerId] = FailureKillCooldown.GetFloat();
+                killer.CustomSyncSettings();//キルクール処理を同期
+            }
+        }
+        public static void OnReportDeadBody()
```
Comment (soukunsandesu) [inline Roles/BountyHunter.cs:77]: https://github.com/tukasa0001/TownOfHost/pull/876/files#diff-52683c9f43eec89a2ec43b1654791089ce6c1ad0b3d0395ed24a55e6afffa1e1R485

File: Roles/BountyHunter.cs [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+            }
+            var target = cTargets[rand.Next(0, cTargets.Count)];
+            Targets[player.PlayerId] = target;
+            Logger.Info($"{player.GetNameWithRole()}のターゲットを{target.GetNameWithRole()}に変更", "BountyHunter");
+
+            //RPCによる同期
+            SendRPC(player.PlayerId, target.PlayerId);
+            return target;
+        }
+        public static void GetAbilityButtonText(HudManager __instance) => __instance.AbilityButton.OverrideText($"{GetString("BountyHunterChangeButtonText")}");
```
Comment (soukunsandesu) [inline Roles/BountyHunter.cs:150]: https://github.com/tukasa0001/TownOfHost/pull/876/files#diff-041a4dcea9b8948ba91c54f339514d6547c974d94b80193b84c70ce616d5d050R90

File: Roles/BountyHunter.cs [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+        public static void DisplayTarget(PlayerControl bounty, TMPro.TextMeshPro LowerInfoText)
```
Comment (soukunsandesu) [inline Roles/BountyHunter.cs:151]: https://github.com/tukasa0001/TownOfHost/pull/876/files#diff-041a4dcea9b8948ba91c54f339514d6547c974d94b80193b84c70ce616d5d050L115

File: Roles/BountyHunter.cs [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+        {
+            var target = GetTarget(bounty);
+            LowerInfoText.text = target == null ? "null" : $"{GetString("BountyCurrentTarget")}:{GetTarget(bounty).name}";
+            LowerInfoText.enabled = target != null || Main.AmDebugger.Value;
+        }
+        public static void AfterMeetingTasks()
```
Comment (soukunsandesu) [inline Roles/BountyHunter.cs:157]: https://github.com/tukasa0001/TownOfHost/pull/876/files#diff-01988ebb33ae4164481da05498d332bc6e5917207043fd7add840d0edfe9d8feL745

**Review by tampopo-dandelion — APPROVED** [review]
特に問題無さそうです

**Pull request merged**